src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts

```diff
@@ -292,23 +305,33 @@ export class ExtensionEditor extends BaseEditor {
 		let recommendationsData = {};
 		if (extRecommendations[extension.id.toLowerCase()]) {
 			addClass(this.header, 'recommended');
-			this.recommendation.textContent = extRecommendations[extension.id.toLowerCase()].reasonText;
+			this.recommendationText.textContent = extRecommendations[extension.id.toLowerCase()].reasonText;
 			recommendationsData = { recommendationReason: extRecommendations[extension.id.toLowerCase()].reasonId };
 		} else {
 			removeClass(this.header, 'recommended');
-			this.recommendation.textContent = '';
+			this.recommendationText.textContent = '';
 		}
 
 		/* __GDPR__
-			"extensionGallery:openExtension" : {
-				"recommendationReason": { "classification": "SystemMetaData", "purpose": "FeatureInsight", "isMeasurement": true },
-				"${include}": [
-					"${GalleryExtensionTelemetryData}"
-				]
-			}
+		"extensionGallery:openExtension" : {
+			"recommendationReason": { "classification": "SystemMetaData", "purpose": "FeatureInsight", "isMeasurement": true },
+			"${include}": [
+				"${GalleryExtensionTelemetryData}"
+			]
+		}
 		*/
 		this.telemetryService.publicLog('extensionGallery:openExtension', assign(extension.telemetryData, recommendationsData));
 
+		const ignoredRecommendations = this.extensionTipsService.getAllIgnoredRecommendations();
+
+		toggleClass(this.header, 'ignored', this.recentlyIgnored.indexOf(extension.id.toLowerCase()) !== -1 || ignoredRecommendations.workspace.indexOf(extension.id.toLowerCase()) !== -1);
+		if (this.recentlyIgnored.indexOf(extension.id.toLowerCase()) !== -1) {
+			this.recommendationText.textContent = localize('recommendationHasBeenIgnoredGlobal', "You have chosen not to receive recommendations for this extension.");
+		}
+		if (ignoredRecommendations.workspace.indexOf(extension.id.toLowerCase()) !== -1) {
+			this.recommendationText.textContent = localize('recommendationHasBeenIgnoredWorkspace', "This extension has been marked as redundant or irrelevant by users of the current workspace.");
```

Comment:
Why does the other message say `You have chosen...` but this one says `by users of the current workspace`?

We dont expect the text to change in real time when some other user marks this as ignore for the workspace

Reply:
As discussed, to always show the `This extension has been marked....` for when a recommendation is ignored for the whole team goes into the category of `disable/black-list extensions for my team` feature which we do not want to get into in this PR/feature.